dist/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Morning To DO List</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <div class="main-list">
+      <li class="todoTitle"><p>Morning To DO List</p></li>
+      <li class="enterdata"><input type="text" class="inputbox" name="newtodo" id="ndwtodo"> <span class="entersymbol">&#9166;</span></li>
+      <ul id="list" class="list">
+      </ul>
+    </div>
```

Comment:
- [ ] Kindly make sure you follow the same design as given in the requirements. You don't have a button at the bottom of the list"`Clear All Completed`". Follow the same design as shown below in the image.

Original design:

<img width="546" alt="Screenshot 2022-06-08 at 3 39 52 PM" src="https://user-images.githubusercontent.com/70269507/172619896-428dd22d-8599-474a-8b0c-7786416c0f76.png">


Your design: 

https://user-images.githubusercontent.com/70269507/172619911-c1777b95-3c78-49f1-9505-ef7d718d9826.mov